### PR TITLE
Added docs for 'get releases' command

### PR DESCRIPTION
**Pull request by AverageMarcus**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/ui-api/kubectl-gs/get-releases.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+```nohighlight
+$ kubectl gs get releases --output yaml
+apiVersion: v1
+kind: List
+items:
+- apiVersion: release.giantswarm.io/v1alpha1
+  kind: Release
+...
+```
```
Comment (marians) [inline src/content/ui-api/kubectl-gs/get-releases.md:103]: Do we do this with other commands? It feels a little too verbose for something common.

Comment (AverageMarcus) [reply src/content/ui-api/kubectl-gs/get-releases.md:103]: I followed the same format as `get nodepools` docs. Happy for advice :)

**Comment by marians** [comment]
Terrific! Could you please also add a link to this page form the "Related" section in https://docs.giantswarm.io/ui-api/gsctl/list-releases/ ?

**Review by marians — APPROVED** [review]

File: src/content/ui-api/kubectl-gs/get-releases.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  main:
+    parent: uiapi-kubectlgs
+aliases:
+  - /reference/kubectl-gs/get-releases/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rainbow
+user_questions:
+  - How can I list releases in a cluster using kubectl?
+  - How can I inspect releases using kubectl?
+last_review_date: 2021-01-01
```
Comment (axbarsan) [inline src/content/ui-api/kubectl-gs/get-releases.md:16]:
```suggestion
last_review_date: 2021-10-04
```

File: src/content/ui-api/kubectl-gs/get-releases.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+---
+
+# `kubectl gs get releases`
+
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, a release pool in this case, or list several of them.
```
Comment (axbarsan) [inline src/content/ui-api/kubectl-gs/get-releases.md:21]:
```suggestion
Like with all `get` commands in `kubectl`, this command can be used to get details on one item, a release in this case, or list several of them.
```

File: src/content/ui-api/kubectl-gs/get-releases.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+To inspect a release's main custom resource in YAML notation, add the `--output yaml` flag (or `-o yaml` in short) to the command.
+
+The following example command would print the main resource for release `v15.2.1`.
+
+```nohighlight
+kubectl gs get releases v15.2.1 --output yaml
+```
+
+When applied without a release name argument, the output will be a list of resources. Example:
```
Comment (axbarsan) [inline src/content/ui-api/kubectl-gs/get-releases.md:93]:
```suggestion
When applied without a release version argument, the output will be a list of resources. Example:
```

**Review by axbarsan — APPROVED** [review]

**Pull request merged**